states/balance_explain.go

```diff
@@ -0,0 +1,248 @@
+package states
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/milvus-io/birdwatcher/models"
+	commonpbv2 "github.com/milvus-io/birdwatcher/proto/v2.2/commonpb"
+	querypbv2 "github.com/milvus-io/birdwatcher/proto/v2.2/querypb"
+	"github.com/milvus-io/birdwatcher/states/etcd/common"
+	etcdversion "github.com/milvus-io/birdwatcher/states/etcd/version"
+
+	"github.com/spf13/cobra"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"google.golang.org/grpc"
+)
+
+const (
+	scoreBasedBalancePolicy               = "score"
+	collectionIDLabel                     = "collection_id"
```

Comment:
I think it would be better to change it to "collection" in order to maintain consistency with the naming conventions of other commands.

Reply:
done